orttraining/orttraining/training_api/module.h

```diff
@@ -34,7 +36,6 @@ struct Parameter {
   // Reset and release the gradient buffer of this Parameter greedily.
   Status ResetGrad();
 
- protected:
```

Comment:
nit: why we remove this?

Reply:
I wasn't sure what purpose it served. What is the intention behind it?